packages/core/admin/admin/src/content-manager/history/components/tests/VersionHeader.test.tsx

```diff
@@ -8,17 +8,32 @@ import { VersionHeader } from '../VersionHeader';
 
 import type { UID } from '@strapi/types';
 
+const useDocumentLayoutMock = jest.fn();
+jest.mock('../../../hooks/useDocumentLayout', () => ({
+  useDocumentLayout: () => useDocumentLayoutMock(),
+}));
+
+const useDocumentMock = jest.fn();
+jest.mock('../../../hooks/useDocument', () => ({
+  useDocument: () => useDocumentMock(),
+}));
```

Comment:
I just did a refactor that means I no longer have to worry about mocking the hook or the requests: I moved the hooks to the page level, and I'm passing down data through the history context.

I prefer it this way because it means having a single place to handle loading and error states. Then in the components like VersionHeader I can assume the data exists. It also makes tests simpler